fluid-project/trivet-monolingual
# feat: remove internationalization features (resolves #4)

## Description

Removes internationalization support, updates docs.

## Steps to test

1. Run: `npm start`
2. Review built pages.

**Expected behavioor:** No errors, no internationalization.

## Additional information

Not applicable.

## Related issues

Resolves #4.
